### DISCUSSION TITLE
增加对译文的进一步处理能力

### DISCUSSION THREAD

**Discussion by violin86318** [immersive-translate/immersive-translate#1009] [Ideas]
### 新功能 | New Feature

1.在每一段译文后增加一个类似 Notion 中块组件，方便对译文进行进一步的处理
比如复制，隐藏，或者切换翻译引擎

### 为什么您想要添加这些功能 | Why you want to add these features

1.方便用户对译文进一步处理的需求
2.方便在页面上针对某一段切换翻译引擎

### 提议 | Proposal

![481697095075_ pic](https://github.com/immersive-translate/immersive-translate/assets/53995897/38d42cc9-97f8-4563-8e70-5e888537026a)
类似截图中的效果

**Comment by theowenyoung** [comment]
这个效果不错～
